core/src/test/java/io/confluent/kafka/schemaregistry/client/MockSchemaRegistryClientTest.java

```diff
@@ -159,5 +163,26 @@ public void testAvroNewProducer() {
     ArrayList<Object> recordList = consume(consumer, topic, objects.length);
     assertArrayEquals(objects, recordList.toArray());
   }
+
+  @Test
+  public void testRegisterAndGetId() throws Exception {
+    MockSchemaRegistryClient client =
+        new MockSchemaRegistryClient(Collections.singletonList(new AvroSchemaProvider()));
+    AvroSchema schema = new AvroSchema("{\"namespace\": \"example.avro\", \"type\": \"record\", " +
+        "\"name\": \"User\"," +
+        "\"fields\": [{\"name\": \"name\", \"type\": \"string\"}]}");
+
+    int id = client.register("test", schema);
+    assertEquals(1, id);
+
+    id = client.register("test2", schema);
+    assertEquals(1, id);
+
+    id = client.getId("test", schema);
+    assertEquals(1, id);
+
+    id = client.getId("test2", schema);
+    assertEquals(1, id);
```

Comment:
Should the test show the incrementing behavior?

Reply:
Done